plugins/system/guidedtours/src/Extension/GuidedTours.php

```diff
@@ -351,12 +354,33 @@ private function processTour($item)
             $temp->tour_id          = $step->tour_id;
             $temp->step_id          = $step->id;
 
-            // Replace 'images/' to '../images/' when using an image from /images in backend.
-            $temp->description = preg_replace('*src\=\"(?!administrator\/)images/*', 'src="../images/', $temp->description);
-
             $tour->steps[] = $temp;
         }
 
         return $tour;
     }
+
+    /**
+     * Return a modified version of a given string with usable image paths for tours
+     *
+     * @param   string  $description  The string to fix
+     *
+     * @return  string
+     *
+     * @since  __DEPLOY_VERSION__
+     */
+    private function fixImagePaths($description)
+    {
+        return preg_replace(
+            [
+                '*src=\"(?!administrator\/)images/*',
+                '*src=\"media/*',
```

Comment:
```suggestion
                '*src="(?!administrator\/)images/*',
                '*src="media/*',
```
Sorry missed this one earlier. Should be ok without the `\`.